gateway/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -1,91 +1,151 @@
-= Anypoint Platform Gateways
+= Flex Gateway Overview
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-An API gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services
-to help you separate orchestration from implementation concerns. You can then leverage the governance capabilities
-of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
+Anypoint Flex Gateway is an Envoy-based, ultrafast lightweight API gateway designed to manage and secure APIs running anywhere. Built to seamlessly integrate with DevOps and CI/CD workflows, Flex Gateway delivers the performance required for the most demanding applications and microservices while providing enterprise security and manageability across any environment.
```

Comment:
Envoy initial cap correct?

Envoy not mentioned in previous descriptions with "ultrafast..."

Reply:
Kim left before the release of Flex, which is why it's not in the Term Reference. I believe I'm being consistent with other products, where we mention the full title (including "Anypoint") at first, and then drop "Anypoint" after that.

Reply:
Envy cap is correct; it's a product title. Added the Envoy mention to previous descriptions.

Reply:
That pattern is correct, but I'm not sure if it's OK legally. Let me check the spreadsheet...Yes, it's fine.
https://docs.google.com/spreadsheets/d/1f6v9cjx84ZAHGeZs36A0clZoQc_ggAe6m6e-NQ9BUuk/edit#gid=71218867

We should add this to the Term Ref.